session/example_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright (c) 2023 VMware, Inc. All Rights Reserved.
```

Comment:
The linter seems to want 2 year values like 

"Copyright (c) 2023-2023 VMware, Inc. All Rights Reserved."